File: net.sf.j2s.ajax/ajaxpipe/net/sf/j2s/ajax/PipeObject.java
package net.sf.j2s.ajax;

public class PipeObject {

	private SimplePipeRunnable pipe;

	public PipeObject(SimplePipeRunnable pipe) {
		super();
		this.pipe = pipe;
	}

	public SimplePipeRunnable getPipe() {
		return pipe;
	}

	public void setPipe(SimplePipeRunnable pipe) {
		this.pipe = pipe;
	}
	
}

File: incubator/net.sf.j2s.test.mauve/j2slib/net/sf/j2s/ajax/PipeObject.js
$_J("net.sf.j2s.ajax");
c$=$_C(function(){
this.pipe=null;
$_Z(this,arguments);
},net.sf.j2s.ajax,"PipeObject");
$_K(c$,
function(pipe){
this.pipe=pipe;
},"net.sf.j2s.ajax.SimplePipeRunnable");
$_M(c$,"getPipe",
function(){
return this.pipe;
});
$_M(c$,"setPipe",
function(pipe){
this.pipe=pipe;
},"net.sf.j2s.ajax.SimplePipeRunnable");
